requirements/deploy.txt

```diff
@@ -369,7 +369,7 @@ snowballstemmer==2.2.0
     # via
     #   -r requirements/pip.txt
     #   sphinx
-sphinx==7.0.1
+sphinx==7.1.1
```

Comment:
I think we deleted the code that requires Sphinx, si se can probably remove it from the requirements